jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/Parser.java

```diff
@@ -639,6 +646,16 @@ public void setIncomingFramesHandler(IncomingFrames incoming)
         this.incomingFramesHandler = incoming;
     }
 
+    public long getMessagesIn()
```

Comment:
Can you add javadoc to make it clear that a message is a frame and not sequence of continuation frames